pkgs/applications/system/howdy/howdy.patch

```diff
@@ -0,0 +1,155 @@
+diff --git a/howdy/src/cli/add.py b/howdy/src/cli/add.py
+index 8951e31..4f793d7 100644
+--- a/howdy/src/cli/add.py
++++ b/howdy/src/cli/add.py
+@@ -30,9 +30,9 @@ import cv2
+ config_path = "/etc/howdy"
+ 
+ # Test if at lest 1 of the data files is there and abort if it's not
+-if not os.path.isfile(config_path + "/dlib-data/shape_predictor_5_face_landmarks.dat"):
++if not os.path.isfile("@PATH@/dlib-data/shape_predictor_5_face_landmarks.dat"):
+ 	print(_("Data files have not been downloaded, please run the following commands:"))
+-	print("\n\tcd " + config_path + "/dlib-data")
++	print("\n\tcd " + "@PATH@/dlib-data")
```

Comment:
Please use more specific name than `PATH`.

Reply:
I hope `@HOWDYPATH@` is fine.

Reply:
It is still a bit ambiguous. Something like `@pkgdatadir@` as per https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Installation-Directory-Variables.html would be best.